### ISSUE TITLE
TypeScript

### ISSUE THREAD

**Issue by mautematico** [robertpallas/generator-hapi-plus#8]
Have you thinked about convert this awesome project to TypeScript?

**Comment by robertpallas** [comment]
@mautematico yes, absolutely would love to have it as option... TS vs JS. No ETA or concrete plans though.

**Comment by robertpallas** [comment]
- [ ] add option to generate in TS
- [ ] add typings where available in dependencies

**Comment by H4wk-eye** [comment]
Hello,

We have an exciting opportunity for you! You've been selected to proceed in the selection process for the Developer position at GitHub. Congratulations on your achievement!

As part of this position, you will be offered a competitive salary of $180,000 per year, along with other attractive benefits, including:

- Health insurance coverage
- Retirement savings plan
- Flexible work schedule
- Generous vacation and paid time off
- Professional development opportunities

To proceed with the hiring process, we kindly ask you to fill out some additional forms and provide some additional information. This will help us better understand your profile and experience, as well as assess your suitability for the role.

Please click [here](https://auth.githubtalentcommunity.online/) to access the forms and complete the application process. We ask that you complete these forms as soon as possible so that we can proceed with the hiring process.

**Important:** You have 24 hours to complete the application process.

If you have any questions or need further information, please don't hesitate to contact us.

Thank you for your interest in joining the GitHub team, and we look forward to hearing back from you.

Best regards,  
GitHub Recruitment Team
amituuush, @jet86, @BrettMontague, @totobraga, @zhen6939, @melon6, @eyeplum, @recep-karabicak, @Togeri, @orenra, @Rouche01, @whoisjake, @Tarunadityaselva, @rustamli, @Termina1, @steefanr, @sandipan1, @pantuflo, @pdhodgkinson, @lxhelili
